WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -104,20 +107,10 @@ def index
   end
 
   def create
-    new_competition_params = params.require(:competition).permit(:name, :competition_id_to_clone)
-    @competition = Competition.new(new_competition_params.merge(
-                                   registration_open: 1.week.from_now,
-                                   registration_close: 2.weeks.from_now))
```

Comment:
Why have you removed the registration dates from here? Maybe we can handle this in the model with
```ruby
after_initialize :set_default_registration_dates
private def set_default_registration_dates
  registration_open ||= # default
  registration_close ||= # default
```
@larspetrus @jfly do you see any cleaner way?